lua/neogit/buffers/commit_select_view/init.lua

```diff
@@ -72,8 +75,14 @@ function M:open(action)
         end
       end,
     },
+    after = function()
+      if commit_at_cursor then
+        vim.fn.search(commit_at_cursor.oid)
```

Comment:
I see, so this will instead go to the line where the hoverer commit appears.

What happens if it doesn't show up in the buffer, most likely due to a different number of commits being selected, as the `Recent Commits` section could load more commits than this

Reply:
I default to 256 commits in the select view, and 10 for the recent commits, so, I guess a user could show 257+ commits in the recent view, but... I doubt they would.